src/modules/blocks/event-datetime/subblocks/content/template.js

```diff
@@ -0,0 +1,253 @@
+			<div
+				key="tribe-editor-event-cost"
+				className="tribe-editor__event-cost"
+			>
+				{ 'prefix' === currencyPosition && <span>{ currencySymbol }</span> }
+				<PlainText
+					className={ classNames( 'tribe-editor__event-cost__value', `tribe-editor-cost-symbol-position-${ currencyPosition }` ) }
+					value={ cost }
+					placeholder={ __( 'Enter price', 'events-gutenberg' ) }
+					onChange={ setCost }
+				/>
+				{ 'suffix' === currencyPosition && <span>{ currencySymbol }</span> }
+			</div>
+		);
+	}
+
+	renderStartDate = () => {
+		const { start, end } = this.props;
+		let startDate = toDate( toMoment( start ) );
+
+		if ( isSameYear( start, end ) && isSameYear( start, TODAY ) ) {
+			startDate = toDateNoYear( toMoment( start ) );
+		}
+
+		return (
+			<span className="tribe-editor__subtitle__headline-date">{ startDate }</span>
+		);
+	}
+
+	renderStartTime = () => {
+		const { start, allDay } = this.props;
+
+		if ( allDay ) {
+			return null;
+		}
+
+		return (
+			<Fragment>
+				{ this.renderSeparator( 'date-time' ) }
+				{ toTime( toMoment( start ), FORMATS.WP.time ) }
+			</Fragment>
+		);
+	}
+
+	renderEndDate = () => {
+		const { start, end, multiDay } = this.props;
+
+		if ( ! multiDay ) {
+			return null;
+		}
+
+		let endDate = toDate( toMoment( end ) );
+
+		if ( isSameYear( start, end ) && isSameYear( start, TODAY ) ) {
+			endDate = toDateNoYear( toMoment( end ) );
+		}
+
+		return (
+			<span className="tribe-editor__subtitle__headline-date">{ endDate }</span>
+		);
+	}
+
+	renderEndTime = () => {
+		const { end, multiDay, allDay } = this.props;
+
+		if ( allDay ) {
+			return null;
+		}
+
+		return (
+			<Fragment>
+				{ multiDay && this.renderSeparator( 'date-time' ) }
+				{ toTime( toMoment( end ), FORMATS.WP.time ) }
+			</Fragment>
+		);
+	}
+
+	renderTimezone = () => {
+		const { setTimeZoneLabel, timeZoneLabel, showTimeZone } = this.props;
+
+		return showTimeZone && (
+			<span
+				key="time-zone"
+				className="tribe-editor__time-zone"
+			>
+				<TimeZone
+					value={ timeZoneLabel }
+					placeholder={ timeZoneLabel }
+					onChange={ setTimeZoneLabel }
+				/>
+			</span>
+		);
+	}
+
+	/**
+	 * Renders a separator based on the type called
+	 *
+	 * @param {string} type - The type of separator
+	 *
+	 * @returns {ReactDOM} A React Dom Element null if none.
+	 */
+	renderSeparator = ( type, className ) => {
+		const { separatorDate, separatorTime } = this.props;
+
+		switch ( type ) {
+			case 'date-time':
+				return (
+					<span className={ classNames( 'tribe-editor__separator', className ) }>
+						{ ' '.concat( separatorDate, ' ' ) }
+					</span>
+				);
+			case 'time-range':
+				return (
+					<span className={ classNames( 'tribe-editor__separator', className ) }>
+						{ ' '.concat( separatorTime, ' ' ) }
+					</span>
+				);
+			case 'all-day':
+				return (
+					<span className={ classNames( 'tribe-editor__separator', className ) }>{ __( 'ALL DAY', 'events-gutenberg' ) }</span>
```

Comment:
I know this was how it was before, but from an accessibility standpoint it would be better to put in "All day" and in CSS `text-transform` it to `uppercase`.

Reply:
That's a much better approach, thanks Paul - I'll put it in